tests/test_async.py

```diff
@@ -67,3 +67,10 @@ async def is_asleep(sleeper):
 async def async_wait_until_awake(sleeper, delay=0.01):
     await wait().at_most(AsyncSleeper.SLEEP_DURATION + 0.1).until_async(partial(sleeper.get_awake_with_delay, delay))
     assert sleeper.awake
+
+
+@pytest.mark.asyncio
+async def test_wait_until_with_lambda_captures_async():
```

Comment:
This test passed without your fix as well

Reply:
Our code does not run on busypie with this lambda provided, it has a runtime error in `_content_of()`, so I'm not sure what you mean by it passes without this fix?

Reply:
fixed